docs/release_notes/0.2.0.md

```diff
@@ -0,0 +1,73 @@
+# Release 0.2.0
+
+## Announcement
+
+This release signifies end of initial development cycle of `eksctl`.
+We have built many new features since first minor release (0.1.0)
+in August 2018, and are switching to a new release cadence now.
+Albeit we have originally planned to do a number of things in 0.2.0,
+the plan has evolved. We will start publishing short-term roadmaps
+in due course for each of planned minor releases.
+
+Please also note that this release has no new features since 0.1.39.
+
+## Acknowledgment
+
+Weaveworks team would like to sencirely thank all of the who have
+contributed to the project, your work is highly appreciated and
+the project would get thus far without you!
+
+We have received many very useful contributions from so many of
+you, some of which came as code, and some as issue or participation
+on Slack, we thank all of you!
+
+Most active contributors in the last 12 month were:
+
+@richardcase @christopherhein @mumoshu @adamjohnson01 @archisgore @Lazyshot @kschumy @toricls @cristian-radu @tedmiston @karinnainiguez @Chabane @tiffanyfay @yutachaos @gruebel @jstrachan @marccarre @PaulMaddox @ozzieba @mukaibot @jmcarp @gotjosh @philoserf @pawelprazak @nckturner @af12066 @mgalgs @Jeffwan @paulbsch @arbourd @cbluth @knorby @gchaincl @mcfedr @overdrive3000 @prageethw @patstrom @sdarwin @unguiculus @Yannig @IPyandy @bowlesns @derwasp @mikeroyal @superseb @zironycho @dcherman @silviogutierrez @sixth @dresnick-sf @jonk @danielfm @denwwer @thapakazi @lucioveloso @callmeradical @manabusakai @justincormack @austbot @pdavies011010 @mreferre @flou @polothy @Tyil @cpaika @arun-gupta @cdenneen @danielchalef @mrichman @whereisaaron @StevenACoffman @procyclinsur @braderhart @aparamon @JasonSwindle @jicowan @olipachi @trondhindenes @milkowski @dingn1 @jamesalbert @stevepe-1 @ArseniiPetrovich @jvanzyl @redborian @tkang007 @elirankon @alexclifford @arielvinas @bnutt @voxxit 
```

Comment:
The names will render with links on the release page, we might want to move the list to the bottom... I'm not too sure. Also, we probably want to break it into multiple lines.

Reply:
Or we could use `<details/>`...

Reply:
Someone broke it into multiple lines.
I've also moved it to the bottom of the notes, the rationale being that the changelog is more critical to readers.